index.html

```diff
@@ -8,22 +8,48 @@
 </head>
 <body>
     <header>
-        <nav class="nav-bar">
-            <div class="left-one">
+        <nav>
+            <div>
                 <i class="fas fa-bars"></i>
                 <i class="fas fa-search"></i>
                 <a class="link-top" href="#" >SPACE & COSMOS</a>
             </div>
-            <div class="center-one">
+            <div>
                 <img src="./src/images/NYLOGO.png">
             </div>
-            <div class="right-one">
-                <button class="top-button">SUSCRIBE NOW</button>
-                <button class="top-button">LOGIN</button>
+            <div>
+                <button >SUSCRIBE NOW</button>
+                <button class="login-btn">LOGIN</button>
             </div>
         </nav>
     </header>
 
+    <main>
+        <div class="container">
+            <div class="feature-img-post">
+                <h1>Space Ripples Reveal Big Bang’s <br> Smoking Gun</h1>
```

Comment:
Your main header could be made bold and centered to better match the article's title.